frontend/src/pages/Artwork/Artwork.tsx

```diff
@@ -112,8 +120,19 @@ export default function Artwork() {
               <button>
                 <Camera />
               </button>
+              <button onClick={() => setShowGroupDrawer(true)}>
+                <Users />
+              </button>
             </div>
           </div>
+          <AnimatePresence>
```

Comment:
I like this component, I haven't seen it used before

Reply:
It's responsible for controlling the drawer entrance and exit animation. If you're interested here's the [link](https://www.framer.com/api/motion/animate-presence/) to the docs